egs2/mls/asr2/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+kmeans_feature=wavlm_large/21  # use model_type/layer_index
+nclusters=1000
+
+src_lang=$(echo "${kmeans_feature}_full_km${nclusters}" | tr "/" "_")
+tgt_lang=en
+
+lang="all" # one of all es en fr nl it pt pl de
+data_split="full" # one of full 1h 10h
+
+train_set="mls_${lang}_train"
+valid_set="mls_${lang}_dev"
+lm_train_text=data/${lang}_lm_train.txt
+
+if [ "$lang" == "all" ]; then
+    for ln in es en fr nl it pt pl de; do
+        test_sets+="mls_${ln}_test "
+    done
+else
+    test_sets="mls_${lang}_test"
+fi
+
+asr_config=conf/train_discrete_asr_e_branchformer1.yaml
+inference_config=conf/decode_ctc0.3.yaml
+
+src_nbpe=3000
+tgt_nbpe=150
+
+# ts: true sequence
+# rm: deduplicated sequence which removes duplicated tokens
+src_case="rm"
+tgt_case="ts"
+
+./asr2.sh \
+    --local_data_opts "--lang ${lang} --data_split ${data_split}" \
+    --portion 1.0 \
+    --kmeans_opts "--batch_bins 4800000" \
+    --kmeans_feature "${kmeans_feature}" \
+    --nclusters "${nclusters}" \
+    --ngpu 1 \
+    --src_lang ${src_lang} \
+    --tgt_lang ${tgt_lang} \
+    --src_token_type "bpe" \
+    --src_nbpe $src_nbpe \
+    --tgt_token_type "bpe" \
+    --tgt_nbpe $tgt_nbpe \
+    --src_case ${src_case} \
+    --tgt_case ${tgt_case} \
+    --speed_perturb_factors "" \
+    --asr_config "${asr_config}" \
+    --inference_config "${inference_config}" \
+    --train_set "${train_set}" \
+    --valid_set "${valid_set}" \
+    --test_sets "${test_sets}" \
+    "$@"
+    # --src_bpe_train_text "data/${train_set}/text.${src_case}.${src_lang}" \
+    # --tgt_bpe_train_text "data/${train_set}/text.${tgt_case}.${tgt_lang}" \
+    # --lm_train_text "data/${train_set}/text.${tgt_case}.${tgt_lang} data/local/other_text/text" \
```

Comment:
```suggestion
```

Reply:
Applied in https://github.com/espnet/espnet/pull/5441/commits/91c2b06069b95492b9f6eee9ffb5e1cc84bdb896 Thanks 👍